snuba/settings_base.py

```diff
@@ -48,6 +48,11 @@
 
 # Processor/Writer Options
 DEFAULT_BROKERS = ['localhost:9093']
+DEFAULT_DATASET_BROKERS = {
+    'events': ['localhost:9093'],
+    'groupedmessage': ['localhost:9093'],
```

Comment:
Shouldn't these not be included in `DEFAULT_DATASET_BROKERS`, since then the `DEFAULT_BROKERS` configuration doesn't apply to these by default?

Reply:
Not sure I follow. My understanding of @JTCunning  request was that we wanted to have multiple bootstrap servers in config to avoid multiple config files and also in order to avoid passing it on the command line. Am I missing something ?

Reply:
Unless we're planning on phasing out `DEFAULT_BROKERS` and requiring `DEFAULT_DATASET_BROKERS` to be defined for each dataset — which doesn't seem like the intention, based on the consumer code change — wouldn't this require anyone who is *not* using `localhost:9093` to explicitly override the default defined here in `DEFAULT_DATASET_BROKERS` for each of these datasets?

For example, this configuration would break [the way `sentry devservices` is configured](https://github.com/getsentry/sentry/blob/7b623edcc89c8cfdb19d98a3d9e5eca3b4b7588b/src/sentry/conf/server.py#L1426). Since `events` and `groupedmessage` are present in the mapping, they won't fall through to the value of `DEFAULT_BROKERS`. If this was just an empty dictionary (which we could add keys to in the production configurations), it'd fall through to the value of `DEFAULT_BROKERS` instead, and work without configuration changes.

I'd argue that `DEFAULT_DATASET_BROKERS` should only be used to override the value of `DEFAULT_BROKERS` if the environment requires it and otherwise should be left empty.

Reply:
ahhh ok, yes sure. I added them here to show how this was used. But indeed they would be added only in the relevant environment.